Makefile

```diff
@@ -280,6 +287,7 @@ undeploy-%: project %-template
 template: \
 	secrets-template \
 	db-template \
+	mq-template \
```

Comment:
did you test this with `make deploy`?

Reply:
Will append test results of `make deploy`.